blender/extension/ui/import_pdb_operator.py

```diff
@@ -58,26 +71,16 @@ def execute(self, context):
 
         return {'FINISHED'}
 
-    def draw_alpha_carbons(self, universe, collection_protein, domain_regions):
+    def draw_alpha_carbons(self, universe, collection_protein, domain_regions, add_convex_hull):
```

Comment:
At some point, it will be useful to add types to all functions to make it clearer how they're supposed to be called. We should use `mypy` to validate them.